### PR TITLE
fix: stop using hardcoded hex colors

### PR DESCRIPTION
**Pull request by adamstankiewicz**
See list of [modified colors](https://docs.google.com/spreadsheets/d/1QBWBMvDUHSD2R6BZoGOuxdanj6Gr19rGPJY4w560p5E/edit?usp=sharing).

### CODE REVIEW

File: src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -4,35 +4,13 @@ footer {
     align-items: center;
 
     .logo {
-      border-right: 1px solid #d4d4d4;
```
Comment (adamstankiewicz) [inline src/components/Footer/Footer.scss:7]: border style moved to the `.border` class

File: src/components/Footer/Footer.scss [hunk 2]
```diff
@@ -4,35 +4,13 @@ footer {
...
-
-  .nav-item {
-    border-right: 1px solid #d4d4d4;
-
-    &:last-child {
-      border-right: 0;
-    }
-
-    .nav-link {
-      padding: 0 1rem;
```
Comment (adamstankiewicz) [inline src/components/Footer/Footer.scss:35]: padding and border styles moved to css classes

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -1,18 +1,11 @@
 header {
-  border-bottom: 1px solid #d4d4d4;
```
Comment (adamstankiewicz) [inline src/components/Header/Header.scss:2]: border style moved to `.border-bottom` css class

File: src/components/Header/Header.scss [hunk 4]
```diff
@@ -1,18 +1,11 @@
...
-
   .navbar-brand {
     img {
       max-height: 60px;
       max-width: 100px;
       width: auto;
     }
   }
-  
-  .dropdown > button {
```
Comment (adamstankiewicz) [inline src/components/Header/Header.scss:12]: user menu button styles moved to `.btn-light` class for now

**Review by raq929 — APPROVED** [review]
I'm a little confused about why package-lock.json changed, but this looks good to me!

**Comment by adamstankiewicz** [comment]
> I'm a little confused about why package-lock.json changed, but this looks good to me!

Ah, yeah I ran `npm i` to after rebasing on `master` and it updated the package-lock.json 🤷‍♂️

**Comment by raq929** [comment]
Heh, I feel like it ought not to be updating it as often now that the packages are mostly locked to one version? But... yeah. npm is mysterious 🤷‍♂️

**Pull request merged**